snaps/openvino/consumer-intel-npu-driver-and-openvino-toolkit-2404/snap/snapcraft.yaml

```diff
@@ -131,6 +129,14 @@ parts:
         echo "${base_path}""${intel_legacy1_icd_so_path}" > "${intel_legacy1_icd}"
       fi
       craftctl default
+  command-chain:
+    plugin: dump
+    source-type: git
+    source: https://github.com/canonical/openvino-toolkit-snap.git
+    #source-branch: openvino-toolkit-2404
+    source-branch: frenchwr/command-chain # temporary for testing
```

Comment:
do you want to wait for this branch to be merged and modify this PR accordingly ?

Reply:
Yes, I will also test pulling directly from the snap itself rather than from GitHub. I think that is my preference.